Fabric.Identity.API/Models/ModelExtensions.cs

```diff
@@ -58,8 +58,10 @@ public static IS4.Client ToIs4ClientModel(this Client client)
                     PostLogoutRedirectUris = client.PostLogoutRedirectUris,
                     EnableLocalLogin = client.EnableLocalLogin,
                     IdentityProviderRestrictions = client.IdentityProviderRestrictions,
-                    LogoutUri = client.LogoutUri,
-                    LogoutSessionRequired = client.LogoutSessionRequired,
+                    FrontChannelLogoutUri = client.LogoutUri,
+                    BackChannelLogoutUri = client.LogoutUri,

```

Comment:
We specify GrantTypes.Implicit or implicit flow above, but we are using BackChannelLogoutUri and LogoutSessionRequired, which is the hybrid flow here.

Reply:
With this one, the LogoutUri wasn't split into Front and Back channel before, so Jason and I decided to assign it to both. We weren't exactly sure which one we were using, and we were thinking it might be easier to put the same value in both to preserve the current functionality. We could remove one if it isn't needed though.